PyIlmBase/CMakeLists.txt

```diff
@@ -55,48 +55,47 @@ endif()
 # for our other dependency - boost.
 # Boost Python has some .. annoyances in that the python module
 # has version names attached to it
+function(PYILMBASE_EXTRACT_REL_SITEARCH varname pyver pyexe pysitearch)
```

Comment:
I bet it's going to be far less work to port Imath to pybind11, than to deal with endless boost::python/cmake curveballs!